crates/client/src/serialize/txt/master.rs

```diff
@@ -159,10 +159,8 @@ impl Parser {
                     match t {
                         // if Dollar, then $INCLUDE or $ORIGIN
                         Token::Include => {
-                            return Err(ParseError::from(ParseErrorKind::Message(
-                                "$INCLUDE is not yet implemented",
-                            )))
-                        }
+                            return Err(ParseError::from(ParseErrorKind::Message("The parser does not support $INCLUDE. Consider inlining file before parsing")))
```

Comment:
This might be better as a panic potentially? Is it a programming error to hit this condition? i.e. the first pass should have expanded all includes, right?

Reply:
It has to be expanded if it was invoked from `FileAuthority`. But the `fn` is `pub`, so it might be used differently, right?

Reply:
sure, we can return the error. I just wondered if we wanted to die quickly. It's fine to leave it as an error.